docs/source/tutorials/transfer_learning/transfer_learning_example.py

```diff
@@ -0,0 +1,247 @@
+from examples.training_scripts.height_example.train_height import (
+    height_config_space,
+    METRIC_ATTR,
+)
+
+from syne_tune import Tuner, StoppingCriterion
+from syne_tune.backend import LocalBackend
+from pathlib import Path
+from syne_tune.optimizer.baselines import BayesianOptimization, ZeroShotTransfer
+from syne_tune.optimizer.schedulers import FIFOScheduler
+
+from syne_tune.optimizer.schedulers.transfer_learning import (
+    TransferLearningTaskEvaluations,
+    BoundingBox,
+)
+
+from syne_tune.optimizer.schedulers.transfer_learning.quantile_based.quantile_based_searcher import (
+    QuantileBasedSurrogateSearcher,
+)
+
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
This would break running it as integration test. You could import this only where it is really used.

Reply:
This is fine as you can install a dependency required for a given example to make it work, for instance see this one:

https://github.com/awslabs/syne-tune/blob/main/.github/workflows/integ-tests.yml#L21

```
  launch_plot_results:
    uses: ./.github/workflows/run-syne-tune.yml
    with:
      additional-command: pip install matplotlib
      script-path: examples/launch_plot_results.py
```

Reply:
Done.